From f455627dabc67636fd01d12b9e84b8df28fb840a Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 14 Feb 2019 18:18:40 +0800
Subject: [PATCH] rk: Makefile: fix Revert "x86: Force asm-goto"

After e08d6de4e532 ("kbuild: remove kbuild cache"), shell-cached is not
work anymore.

Fixes: d9e69500bc91 ("Revert "x86: Force asm-goto"")
Change-Id: Id6a4bdaacc8c2f1d5ace566941799d1e352bc028
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index cb305dab5322..602718f9ec35 100644
--- a/Makefile
+++ b/Makefile
@@ -694,7 +694,7 @@ KBUILD_CFLAGS += $(call cc-ifversion, -lt, 0409, \
 KBUILD_CFLAGS	+= $(call cc-option,--param=allow-store-data-races=0)
 
 # check for 'asm goto'
-ifeq ($(call shell-cached,$(CONFIG_SHELL) $(srctree)/scripts/gcc-goto.sh $(CC) $(KBUILD_CFLAGS)), y)
+ifeq ($(shell $(CONFIG_SHELL) $(srctree)/scripts/gcc-goto.sh $(CC) $(KBUILD_CFLAGS)), y)
 	KBUILD_CFLAGS += -DCC_HAVE_ASM_GOTO
 	KBUILD_AFLAGS += -DCC_HAVE_ASM_GOTO
 endif
-- 
2.35.3

